py/private/py_binary.bzl

```diff
@@ -172,7 +173,18 @@ python = use_extension("@rules_python//python/extensions:python.bzl", "python")
 python.toolchain(python_version = "3.8.12", is_default = False)
 python.toolchain(python_version = "3.9", is_default = True)
 ```
-"""
+""",
+    ),
+    "package_collision": attr.string(
```

Comment:
🧵 Naming?

Should we allow this to be set via a Starlark flag globally?

Reply:
Starlark flags discard the analysis cache, so I think we try to avoid. I think the common case is that one venv has a collision but others in that repository do not? In that case I think it's okay to mark it on each one.